### ISSUE TITLE
Content of database

### ISSUE THREAD

**Issue by gdutoo** [Frank-MandarinFish/essda-public#1]
Fabrics in the Dickson database of the Beta version are not the same fabrics than in current official database.
It looks like the Beta has been built from an old version.

**Comment by Frank-MandarinFish** [comment]
That is correct. The dev database was imported several month ago and does not reflect the current state of the live database. You can play around with it as you desire, there will be a fresh import before v3 goes live.

**Comment by gdutoo** [comment]
OK ; I just wanted to be sure.
I go on with evaluation.
Thank you
KR
Gonzague

De : SykNet ***@***.***>
Envoyé : mardi 3 décembre 2024 17:27
À : Frank-MandarinFish/essda-public ***@***.***>
Cc : DUTOO, Gonzague ***@***.***>; Author ***@***.***>
Objet : Re: [Frank-MandarinFish/essda-public] Content of database (Issue #1)


That is correct. The dev database was imported several month ago and does not reflect the current state of the live database. You can play around with it as you desire, there will be a fresh import before v3 goes live.
